cpp/daal/src/algorithms/dtrees/forest/classification/oneapi/df_classification_train_hist_kernel_oneapi.h

```diff
@@ -170,8 +171,13 @@ class ClassificationTrainBatchKernelOneAPI<algorithmFPType, hist> : public daal:
     const size_t _maxBins                 = 256;
     const size_t _reduceLocalSizePartHist = 64;
 
+    const size_t _minPreferableLocalSizeForPartHistKernel = 32;
```

Comment:
If it is related only to one kernel, isn't it better to make this a local variable inside the kernel?

Reply:
this value is used in internal kernel (not the one located on first line of call) and depends on number of features, therefore it is better to calculate it only one time at start and use the stored value